README.md

```diff
@@ -1491,7 +1491,7 @@ Here's the result:
 
 ## Configuration
 
-You can adjust the behavior of the menu builder in `config/settings.php` file. Currently it provide a few options out of the box:
+You can adjust the behavior of the menu builder in `config/laravel-menu.php` file. Currently it provide a few options out of the box:
```

Comment:
This file has not changed in master. It is still `config/settings.php`

I think this PR would confuse people?